lib/pluginmanager/update.rb

```diff
@@ -34,14 +37,15 @@ def update_gems!
     # remove any version constrain from the Gemfile so the plugin(s) can be updated to latest version
     # calling update without requiremend will remove any previous requirements
     plugins = plugins_to_update(previous_gem_specs_map)
-    plugins
-      .select { |plugin| gemfile.find(plugin) }
-      .each { |plugin| gemfile.update(plugin) }
+    filtered_plugins = plugins.collect { |plugin| gemfile.find(plugin) }
+      .compact
+      .reject { |plugin| plugin.options.has_key?(:path) }
```

Comment:
what about plugins using a :git option ? this is actually nice when we want to build test with remote repos :+1: